explore-lr-experiment--1/run_0/run-explore-lr.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
I also am not sure whether these run scripts are really necessary. They're automatically generated by the parallel run script, so it seems a little redundant.

Reply:
These were unintended. Will delete them, thanks